tiedonsiirtoprotokollan_muutoshistoria.md

```diff
@@ -1,3 +1,11 @@
+## 13.5.2022
```

Comment:
Voisi kysäistä asiantuntijoilta, haluavatko näitä Kelan tai muun API:n tietomallin muutoksia dokumentoitavan tänne.

Oma käsitykseni on ollut, että tähän tiedostoon tulisi vaan pää-scheman muutokset, joista oppilashallintojärjestlmien kehittäjät ovat kiinnostuneita. APIt on dokumentoitu (toistaiseksi ilman muutoshistoriaa) `rajapinnat_luovutuspalvelu.md`hen.

Reply:
Totta, hyvä huomio!